### DISCUSSION TITLE
Can't upload while applying in github student developer pack

### DISCUSSION THREAD

**Discussion by tungtungsahur25** [community/community#19416] [GitHub Education] [answered]
I've answered all questions correctly but I can't upload my proof there. Only take image button but when I submit my proof by taking picture on it, It auto decline my application. Thank you for those who will help. 😊

**Comment by melvincwng** [comment, marked as answer]
@CarlR0001, please kindly raise a ticket to [Github Education Support](https://support.github.com/contact/education) for them to investigate this issue.

They usually would respond back within a couple of working days. Please kindly wait for their reply.

If this is helpful, please 'Mark as answer' so that others know that this issue is addressed.

Thank you and have a great day ahead 😊

**Reply by tungtungsahur25** [reply]
Crated the ticket and will wait for response. Thank you

**Reply by kykykyl2** [reply]
> @CarlR0001, veuillez ouvrir un ticket auprès [du support Github Education](https://support.github.com/contact/education) pour qu'ils enquêtent sur ce problème.
> 
> Ils vous répondront généralement dans un délai de quelques jours ouvrables. Veuillez attendre leur réponse.
> 
> Si cela est utile, veuillez « Marquer comme réponse » afin que les autres sachent que ce problème est résolu.
> 
> Merci et bonne journée à vous 😊

hello i have the same problem but ticket for education can't be create help me pls

**Comment by Pengwei-Jin** [comment]
Hi, did you get a reply to your ticket? I opened a ticket three days ago but haven't got any reply.

**Reply by tungtungsahur25** [reply]
No. Haven't received any replies or updates :(

**Reply by melvincwng** [reply]
@CarlR0001 and @Pengwei-Jin, hi there please kindly wait for the support engineers at GitHub to get back to you.

They require some time to investigate the issue before replying to your tickets in GitHub Support.

Hope this clarifies.

In the meantime, sit back, relax and keep on coding :).

If you would like to stay in touch, kindly follow me on GitHub.

Thanks, and have a good day ahead.

**Reply by Pengwei-Jin** [reply]
Thank you for your reply. : ) I will wait for another several days.

**Reply by ahmetqunay** [reply]
How did you solve the problem?

**Comment by JDO-exe** [comment]
Did you get a reply by now?

**Comment by sahilkorde121** [comment]
there's no option showing to upload the proof image of student ID card while filling the verification form for student developer pack and only taking picture through webcam is showing. Some one plzz help me to resolve this issue

**Reply by P10swati** [reply]
Could you apply for it ?

**Reply by ahmetqunay** [reply]
How did you solve the problem?
